components/x-gift-article/storybook/free-article.js

```diff
@@ -3,7 +3,9 @@ const nonGiftArticleUrl = `${articleUrl}?shareType=nongift`
 
 exports.args = {
 	title: 'Share this article (free)',
+	isRafActive: false,
 	isFreeArticle: true,
+	raf: { title: 'Custom title' },
```

Comment:
if `isRafActive: false,`
`raf: { title: 'Custom title' },` would do nothing?

Should it be:
if `isRafActive: true,`
`raf: { title: 'Custom title' },`

Reply:
Args can be changed in storybook, just by having it here as false, it can be toggled later.